src/components/Card/Card.theme.ts

```diff
@@ -89,7 +89,7 @@ export const CardStyle = {
           md: 1,
         },
         borderTopWidth: {
-          base: 1,
+          base: 0,
```

Comment:
`dashboard` card style, border-top-width for base (mobile) does not apply anymore or that will look ugly with a double border-top now that dashboard is gone on mobile